docs/Developers/Writing_Good_Commits.rst

```diff
@@ -83,8 +83,10 @@ Example:
    -  **Short Description**
 
       - Starts with a capital letter.
-      - Written in imperative present tense (i.e. ``Add something``, not
-        ``Adding something`` or ``Added something``).
+      - Written in `present tense imperative mood
+        <https://en.wikipedia.org/wiki/Imperative_mood>`_.
```

Comment:
I think adding link only to `imperative mood` will be good. And maybe you could use [this](http://docutils.sourceforge.net/docs/user/rst/quickref.html#external-hyperlink-targets) syntax.